IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/QuantitySets/Qto_PictorialSignQuantities/DocQuantitySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_PictorialSignQuantities" UniqueId="c0785096-718a-459d-8ae8-b76089be6d06">
+	<Quantities>
+		<DocQuantity xsi:nil="true" href="Area_1CnNsFYBT6AemE3IaDxL7" />
+		<DocQuantity xsi:nil="true" href="SignArea_32i2POqiz7sxewNKXzANau"/>
```

Comment:
Height and width of interest?

Reply:
that is in the entity qto  `Qto_SignBaseQuantities` these are additional qtos for pictoral signs as the entity includes markers which are just posts

Reply:
I see.